src/components/tl/ucp/tl_ucp.c

```diff
@@ -108,6 +108,11 @@ static ucc_config_field_t ucc_tl_ucp_lib_config_table[] = {
      ucc_offsetof(ucc_tl_ucp_lib_config_t, scatter_kn_radix),
      UCC_CONFIG_TYPE_UINT},
 
+    {"REDUCE_AVG_PRE_OP", "1",
+     "Reduce will perform division by n in result, else - in earlier stages",
```

Comment:
@lappazos is the param description correct? Looks like vice versa: if 0 then reduce will perform division in result. also lets rephrase: instead of "division by n" lets explicitly say "division by team_size "